### PR TITLE
Create 2014-01-22-Jacob'sTurtlePicAndPythonExercises

### PR DESCRIPTION
**Pull request by ghost**
http://jacobthill.github.io/spring2014/

### CODE REVIEW

**Comment by eah13** [comment]
@jacobthill please always provide a link to your site to verify that it's working.  You can also look over closed pull requests to see examples of what others have done.

**Comment by eah13** [comment]
Here's why it's important to view and provide the link: your post isn't showing up
http://jacobthill.github.io/spring2014/

First of all it needs a `md` extension on it to render properly. The apostrophe in the filename will confuse Jekyll too.  

The URL you have embedded doesn't lead to a `.jpg` or `.png`, which will prevent it from rendering. You'll need to right click the image and open it in its own window until you get the raw image.

Finally, it seems you need to update the title- did you copy/paste?  That's OK to do, but it's very important to doublecheck that everything you've pasted in is correct!

**Comment by gerbal** [comment]
@jacobthill your username in the header of your post needs to be the same as it is in [config.yml](https://github.com/silshack/spring2014/blob/gh-pages/_config.yml#L214)

**Comment by gerbal** [comment]
@jacobthill also facebook images probably won't show up reliably. try using http://imgur.com

File: _posts/2014-01-22_JacobTurtle.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
++---
```
Comment (eah13) [inline _posts/2014-01-22_JacobTurtle.md:1]: This entire file should be deleted as it seems like a duplicate.  You should be able to view and then delete it relatively easily.

File: _posts/2014-01-17-JacobBasicPythonExercise.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
```
Comment (eah13) [inline _posts/2014-01-17-JacobBasicPythonExercise.md]: This file needs to be renamed to have the correct date.  You can rename file after clicking 'edit'.

**Comment by eah13** [comment]
@jacobthill excellent work on that Turtle post!  it's a very nice picture you've drawn.

The Python exercises look good too but there are still a few problems with the posts you're submitting.  See me comments in the code.

**Comment by eah13** [comment]
Alright- looks good to go.

**Pull request merged**